lib/open_mic.rb

```diff
@@ -0,0 +1,19 @@
+class OpenMic
+  attr_reader :info, :location, :date, :performers
+
+  def initialize(info)
+    @info = info
```

Comment:
No need for the @info to store the hash, because you deconstructed the hash with the following two instance variables!